src/network/networkd-radv.c

```diff
@@ -27,29 +40,22 @@ int config_parse_router_prefix_delegation(
                 void *userdata) {
 
         Network *network = userdata;
-        int d;
+        RADVPrefixDelegation d;
 
         assert(filename);
         assert(section);
         assert(lvalue);
         assert(rvalue);
         assert(data);
 
-        if (streq(rvalue, "static"))
-                network->router_prefix_delegation = RADV_PREFIX_DELEGATION_STATIC;
-        else if (streq(rvalue, "dhcpv6"))
-                network->router_prefix_delegation = RADV_PREFIX_DELEGATION_DHCP6;
-        else {
-                d = parse_boolean(rvalue);
```

Comment:
I don't think we can just replace this with "yes" and "no" without breaking backwards compatibility. `parse_boolean` also takes `1`, `y`, `true`, `0`, `n`, `false` and so on. In theory, it's possible to update `radv_prefix_delegation_table`, but it would be hard to keep up with `parse_boolean` should anyone decide to extend it in the future. I'd leave it as is.

Reply:
I thought that `DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN` treats such words correctly... But I have not tested that on this setting. I will test them.

Reply:
I'm sorry. I totally missed the `WITH_BOOLEAN` part.